AzureRmStorageTableCoreHelper.psm1

```diff
@@ -595,6 +609,15 @@ function Get-AzTableRow
 		$TableQuery.FilterString = $Filter
 	}
 
+	# Selecting columns if specified
+	if ($null -ne $SelectColumns){
+		[string[]]$Cols = $SelectColumns
+		$ColumnsList = New-Object System.Collections.Generic.List[string]
+		$ColumnsList.AddRange($Cols)
+
+		$TableQuery.SelectColumns = $ColumnsList

```

Comment:
@PaulCheng , I believe this will not have any effect since all handling is done on client side correct?

Reply:
The column selection becomes part of the table query and is reflected in the $select parameter of the API call to table storage. When using this, I only receive back the columns I need in the response body, instead of receiving all columns if I do not use this parameter.

In my specific use case, I have a large table with many columns, but only need a subset of columns for a specific query. Using this method significantly improves query response time.

You can verify this by running fiddler with and without the -selectcolumns parameter.

Without -SelectColumns:
![image](https://user-images.githubusercontent.com/12546649/108571888-f46f7100-72c5-11eb-85d0-4a10deca3707.png)

With -SelectColumns:
![image](https://user-images.githubusercontent.com/12546649/108571895-f89b8e80-72c5-11eb-9f1a-259436bd9209.png)

Reply:
Thanks for the confirmation, let me review and get back with comments.

Reply:
Sounds good, thanks for building this module, its been a huge help to us!